Client/Ecosia/UI/NTP/Impact/ClimateImpactInfo.swift

```diff
@@ -112,21 +95,10 @@ enum ClimateImpactInfo: Equatable {
             "total_invested_image"
         }
     }
-    
-    var progressIndicatorValue: Double? {
-        switch self {
-        case .search:
-            return User.shared.progress
-        case .referral, .totalInvested, .totalTrees:
-            return nil
-        }
-    }
-    
+        
     /// Created to be used for comparison without taking the associated types arguments into consideration.
     var rawValue: Int {
         switch self {
-        case .search:
-            return 0
```

Comment:
Given this is used for comparison, shouldn't we update the others so it starts on `0`? I guess this is minor and not a blocker, even wondering if there is a better way to do it 🤔